### PR TITLE
A small collection of fixes

### PR DESCRIPTION
**Pull request by mattcan**
Two main things in here:

* Solved #34 by setting a default object on `lowerCaseKeys` because it is possible to send in an event without headers. In production, we saw this occurring with `serverless-warmup` plugin sending no headers.
* Continued the work in #18 by fixing the tests and updating a missed `done()` usage

(High-fives to @maumercado for finding and suggesting the fix in `lowerCaseKeys`)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -59,10 +65,10 @@ module.exports.serverless = appFn => {
 
     // Bail for null body
     if (!event.body) {
-      return context.done(null, {
```
Comment (maumercado) [inline index.js:62]: This is the only part Im not sure of, can you clarify why the callback was removed?

**Review by maumercado — CHANGES_REQUESTED** [review]
one question only

File: index.js (hunk 1 above)
Comment (mattcan) [reply index.js:62]: See #18, apparently it is no longer in use. It was just missed in that pr so I doubled up some work

File: tests/index.test.js [hunk 2]
```diff
@@ -44,10 +49,35 @@ describe('serverless-lambda', () => {
...
+        'x-github-delivery': 123
+      }
+    }
+
+    const result = await handler(event, context)
+    expect(spy).not.toHaveBeenCalled()
+    expect(result.statusCode).toBe(400)
+  })
+
+  it('responds with a 400 when no headers are present', async () => {
```
Comment (gr2m) [inline tests/index.test.js:72]: I like adding a reference to an issue/pull request so we don't have to wonder later why this exists

```suggestion
  it('responds with a 400 when no headers are present (#48)', async () => {
```

**Review by maumercado — APPROVED** [review]
LGTM with @gr2m suggestion

File: tests/index.test.js (hunk 2 above)
Comment (mattcan) [reply tests/index.test.js]: Makes sense to me!

**Review by gr2m — APPROVED** [review]
Looks great, thank you!

**Pull request merged**